RELEASE_NOTES.md

```diff
@@ -1,5 +1,21 @@
 <!-- markdown-link-check-disable -->
 
+# 0.57.0rc1
+
+This release candidate brings in a support for the fully new ZenML Dashboard experience, 
```

Comment:
```suggestion
This release candidate introduces support for the fully new ZenML Dashboard experience, 
```

Reply:
I will commit this once CI is completed.
Before that can I get your LGTM not to bother you out of business hours?